packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -746,25 +746,25 @@ export class MgtPerson extends MgtTemplatedComponent {
     switch (availability) {
       case 'Available':
         switch (activity) {
-          case 'Available':
-            presenceIcon = getSvg(SvgIcon.PresenceAvailable);
-            break;
           case 'OutOfOffice':
             presenceIcon = getSvg(SvgIcon.PresenceOofAvailable);
             break;
+          // OutOfOffice and Uknowns
+          default:
```

Comment:
Let's be explicit about the items that are being bucketed into the default

```suggestion
          case 'Available':
          default:
```